### PR TITLE
bug fix of the download failed in offline and program name in details page

### PR DESCRIPTION
**Pull request by farhanp1502**

### CODE REVIEW

File: projects/project-player-library/src/lib/pages/details-page/details-page.component.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div class="flex justify-between">
     <span class="details-card-header">
         <h4 class="break-all">{{projectDetails?.title || "Undefined"}}</h4>
+        <p class="pl-2 break-all text-base text-gray-500">{{projectDetails?.programName}}</p>
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/pages/details-page/details-page.component.html]: Is it the same styling which is used in the preview-page?

Comment (farhanp1502) [reply projects/project-player-library/src/lib/pages/details-page/details-page.component.html]: yes
padding i have added

Comment (Vinod-V3) [reply projects/project-player-library/src/lib/pages/details-page/details-page.component.html]: is that required?

Comment (farhanp1502) [reply projects/project-player-library/src/lib/pages/details-page/details-page.component.html]: removed it

**Review by Vinod-V3 — APPROVED** [review]

**Pull request merged**